data_steward/tools/add_hpo.py

```diff
@@ -140,6 +179,60 @@ def add_hpo_site_mappings_csv(hpo_id,
                        index=False)
 
 
+def add_hpo_id_bucket_name_csv(hpo_id, bucket_name, hpo_id_bucket_name_path):
+    """
+    Writes df with hpo_id and bucket_name to the hpo_id_bucket_name config file
+
+    :param hpo_id: hpo_ identifier
+    :param bucket_name: GCS bucket name for the site.
+    :param hpo_id_bucket_name_path: path to csv file containing hpo site information
+    :return:
+    """
+    hpo_file_df = add_hpo_id_bucket_name_file_df(hpo_id, bucket_name,
+                                                 hpo_id_bucket_name_path)
+    hpo_file_df.to_csv(hpo_id_bucket_name_path,
+                       quoting=csv.QUOTE_ALL,
+                       index=False)
+
+
+def add_hpo_site_to_csv_files(hpo_id,
+                              hpo_name,
+                              org_id,
+                              bucket_name,
+                              hpo_site_csv_path,
+                              display_order=None):
+    """
+    Update both csv files from devops to include data for the newly added site.
+
+    :param hpo_id: hpo_ identifier
+    :param hpo_name: name of the hpo
+    :param org_id: hpo organization identifier
+    :param bucket_name: GCS bucket name of the site
+    :param hpo_site_csv_path: path to csv file containing hpo site information
+    :param display_order: index number in which hpo should be added in table
+    :return:
+    """
+    # Check if csv files exist in given path.
+    hpo_site_mappings_path = hpo_site_csv_path + '/hpo_site_mappings.csv'
+    hpo_id_bucket_name_path = hpo_site_csv_path + '/hpo_id_bucket_name.csv'
```

Comment:
These two lines will cause a bad path if `hpo_site_csv_path` already has a trailing slash (e.g. combined path can become "hpo_site_csv_path//hpo_id_bucket_name.csv". I recommend using pathlib to form the paths.
```suggestion
    hpo_site_mappings_path = Path(hpo_site_csv_path) / 'hpo_site_mappings.csv'
    hpo_id_bucket_name_path = Path(hpo_site_csv_path) / 'hpo_id_bucket_name.csv'
```

Reply:
sure.